terraform/main.tf

```diff
@@ -36,6 +36,16 @@ resource "google_storage_bucket" "tfstate" {
   }
 }
 
+resource "google_storage_bucket" "well_gas_pdfs" {
+  name          = "${random_id.bucket_prefix.hex}-catalyst-coop-well-gas-pdfs"
```

Comment:
To make this `well-gas-pdfs.catalyst.coop` I would have to verify ownership of that domain, which seemed like a pain. So I just stuck with this bucket prefix.

Reply:
You personally would need to verify ownership? We've verified ownership of `catalyst.coop` already. I thought this was a project-level thing...

Reply:
I think we'd need to verify ownership of the subdomain too - at least that's what the instructions seemed to imply.